### PR TITLE
fix lint error in client verify process

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Fix lint error in client verify process

### CODE REVIEW

File: client/quantum_serverless/core/provider.py [hunk 1]
```diff
@@ -321,14 +321,6 @@ def __init__(
             available_compute_resources: available clusters in provider
         """
         super().__init__(name)
-        warnings.warn(
-            "`KuberayProvider` is deprecated "
-            "and will be removed in v0.3. "
```
Comment (psschwei) [inline client/quantum_serverless/core/provider.py:326]: I don't think we should drop this before v0.3, probably better to have pylint ignore it via something like `# pylint: disable=undefined-variable`

Comment (akihikokuroda) [reply client/quantum_serverless/core/provider.py:326]: @psschwei thanks for review.  It looks like this should have forgotten to  take out as a part of #697.

Comment (psschwei) [reply client/quantum_serverless/core/provider.py:326]: good catch -- yes, we should add that import back

Comment (akihikokuroda) [reply client/quantum_serverless/core/provider.py:326]: OK.  I'll do that.  Thanks.

**Review by psschwei — APPROVED** [review]
LGTM

**Pull request merged**